packages/pipeline-editor/src/CustomFormControls/CustomFieldTemplate.tsx

```diff
@@ -14,22 +14,21 @@
  * limitations under the License.
  */
 
-import { FieldTemplateProps } from "@rjsf/core";
+import { FieldTemplateProps } from "@rjsf/utils";
 
 export const CustomFieldTemplate: React.FC<FieldTemplateProps> = (props) => {
   let children = props.children;
-  if (props.uiSchema["ui:field"] === "hidden") {
+  if (props.uiSchema?.["ui:field"] === "hidden") {
     return <div />;
   }
-  if (props.uiSchema["ui:readonly"]) {
+  if (props.uiSchema?.["ui:readonly"]) {
     children = (
       <div style={{ paddingTop: "8px" }}>
         {props.formData ?? props.schema.default}
       </div>
     );
   }
-  const requiredError = props.required && props.formData === undefined;
-  const hasError = props.rawErrors || requiredError;
+  const hasError = (props.rawErrors?.length ?? 0) > 0;
```

Comment:
would `const hasError = !!props.rawErrors?.length` give the same result in simpler code?

Reply:
Would that check for rawErrors having length 0?

Reply:
Ohh wait ok I think I see what you're saying

Reply:
it should work since 0 is falsy